extension/js/inspector/app/modules/Data/views/Layout.js

```diff
@@ -89,12 +93,24 @@ define([
       }
     },
 
+    showClassCollectionModels: function(opt) {
+      var filteredList = this.options.modelCollection.filter(function(model) {
+        return model.get('classId') === opt.classId
+      });
```

Comment:
@jasonLaster Are you talking about this check?

```
var filteredList = this.options.modelCollection.filter(function(model) {
  return model.get('classId') === opt.classId
});
```

Turns out I can't move it since I will need the click events to know which model the user clicked on.

```
dataCommands: {
 'show:classCollectionModels': 'showClassCollectionModels'
},
```